src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -53,6 +54,7 @@ private void setSubmissionData(FilingApi filing, OverseasEntitySubmissionDto sub
         Map<String, Object> data = new HashMap<>();
         data.put(ENTITY_FIELD, submissionDto.getEntity());
         data.put(BENEFICIAL_OWNERS_INDIVIDUAL_FIELD, submissionDto.getBeneficialOwnersIndividual());
+        data.put(BENEFICIAL_OWNERS_GOVERNMENT_OR_PUBLIC_AUTHORITY, submissionDto.getBeneficialOwnersGovernmentOrPublicAuthority());
```

Comment:
what : BENEFICIAL_OWNERS_GOVERNMENT_OR_PUBLIC_AUTHORITY
why:  needs to be covered by test
where: testFilingGenerationWhenSuccessful() in FilingServiceTest. The test has some asserts to check the other data in the FilingApi, looks like we should also assert the BENEFICIAL_OWNERS_GOVERNMENT_OR_PUBLIC_AUTHORITY data as well.

Reply:
I see that's been done.